demo/parameter_files/pmp_input_parameters_demo1.py

```diff
@@ -1,8 +1,11 @@
 ################################################################################
-#  SAMPLE INPUT PARAMETER FILE FOR THE PCMDI METRICS PACKAGE (PMP V1.1)
+#  THIS IS A DEMO INPUT PARAMETER FILE FOR THE PCMDI METRICS PACKAGE (PMP V1.1)
 #  THIS IS A VERY SIMPLE EXAMPLE, ONLY COMPUTING STATISTICS FOR ONE MODEL VERSION AND ONE VARIABLE 
 #  SAMPLE MODEL AND OBSERVATIONAL DATA TO RUN THIS CODE ARE AVAILABLE FROM: 
-# 
+#  http://oceanonly.llnl.gov/gleckler1/pmp-demo-data/pmpv1.1_demodata.tar
```

Comment:
@gleckler1 I think the demo should do this for them, or at least try to. Do you want me to set this up?